pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -402,6 +402,13 @@ func (s *ServiceImpl) buildDashboardNavLinks(c *contextmodel.ReqContext, hasEdit
 				Text: "New dashboard", Icon: "plus", Url: s.cfg.AppSubURL + "/dashboard/new", HideFromTabs: true, Id: "dashboards/new", ShowIconInNavbar: true, IsCreateAction: true,
 			})
 		}
+
+		if hasAccess(hasEditPermInAnyFolder, ac.EvalPermission(dashboards.ActionDashboardsCreate)) {
```

Comment:
This is the exact same if as above no? Can we just merge them?

Reply:
oh yeah, idk how i missed that! 😅